ui/apps/platform/src/Containers/MainPage/Sidebar/NavigationSidebar.tsx

```diff
@@ -122,6 +125,33 @@ function NavigationSidebar({
                     path={complianceBasePath}
                     title={basePathToLabelMap[complianceBasePath]}
                 />
+
+                {isFeatureFlagEnabled('ROX_VULN_MGMT_WORKLOAD_CVES') &&
+                    isFeatureFlagEnabled('ROX_POSTGRES_DATASTORE') && (
+                        // TODO We need to designate this as Tech Preview in a more standard way, based on UX guidance
```

Comment:
UX is working on a consistent way to designate something as a tech preview in the UI, so for now, I just included plain text "(preview)" in the navigation.

This gets the point across, and is ugly enough that we won't forget to change it.

Reply:
As a side note, `<NavExpandable>` only accepts a `string` for the `title` prop, unlike `<LeftNavItem>` which accepts a `string | ReactNode`. The PatternFly source renders the title directly as a child though, so there is no technical reason we couldn't pass a component for the `title`.

If we want to display the tech preview in the same way we are for the Network Graph 2.0, we will either need to do some type-casting or reach out to the PF team and see if the prop type can be updated.

Reply:
String seems safer and simpler.

Decorating every little detail slows us down and probably slows down visual processing of users too.